.vscode/cspell.json

```diff
@@ -16,9 +16,18 @@
     "openflights",
     "parens",
     "prettierrc",
+    "redrive",
     "slsw",
     "vite"
   ],
-  "ignoreWords": ["checkedin", "laxlga", "youre", "iatafilter", "mweb", "requestfinished"],
+  "ignoreWords": [
+    "checkedin",
```

Comment:
what do these do?

Reply:
These are words that are not really words but that we want to ignore with cspell. cspell will not consider them misspelled, but will also not make suggestions based on them.